src/index.html

```diff
@@ -1,7 +1,7 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
```

Comment:
The `<html>` tag is incorrectly self-closed. Please remove the `/` at the end of the `<html>` tag to fix this issue.